src/noos_inv/docker.py

```diff
@@ -13,6 +13,7 @@
         "user": "AWS",
         "token": None,
         "name": "webserver",
+        "file": None,
```

Comment:
Across the repo, only "sensitive" configs are left nullable (or `None`) and to be defined by OS environnement variables. Following this convention, the `file` config could be set at Docker's default `./Dockerfile`.